### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by yuetlin**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by yuetlin — COMMENTED** [review]
ok!

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
```
Comment (123456789pyl) [inline index.html:49]: suggestion

File: index.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
 
-<h2 hidden>Game over</h2>
```
Comment (123456789pyl) [inline index.html:50]: <h2 hidden>Game over!</h2>

File: index.html [hunk 3]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
...
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (yuetlin) [inline index.html]:
```suggestion
<h2 hidden>Game over! Refresh to play again 🧑‍💻 🤖!</h2>
```

**Pull request merged**